orca-core/src/main/groovy/com/netflix/spinnaker/orca/RetrySupport.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2017 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca;
+
+import java.util.function.Supplier;
+
+public class RetrySupport {
```

Comment:
A candidate for `kork`? We have a similar Retry class in `clouddriver` (original inspiration).

Reply:
seems reasonable to me